index.html

```diff
@@ -180,50 +195,41 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
                     </div>
+                   <div class="box-p">
                     <p class="text-white">An Open Source organization focuses on Data Structure and Algorithms.</p>
                 </div>
-                <div class="col-lg-2 col-md-2 col-sm-6 col-12 offset-lg-1">
+                </div>
+                <div class="col-lg-2 col-md-2 col-sm-6 col-12 offset-lg-1 container-useful-links">
                     <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Useful Links</h5>
                     <ul>
-                        <a class="text-white mb-lg-3" href="#about">About</a><br>
-                        <a class="text-white mb-lg-3" href="./team.html">Our Team</a><br>
-                        <a class="text-white mb-lg-3" href="#footer">Contact</a><br>
-                        <a class="text-white mb-lg-3" href="./sitemap.html">SiteMap</a>
+                        <a class="mb-lg-3" href="#about">About</a><br>
+                        <a class="mb-lg-3" href="./team.html">Our Team</a><br>
+                        <a class="mb-lg-3" href="#footer">Contact</a><br>
+                        <a class="mb-lg-3" href="./sitemap.html">SiteMap</a>
                     </ul>
                 </div>
-                <div class="col-lg-3 col-md-3 col-sm-6 col-12">
+                <div class="col-lg-3 col-md-3 col-sm-3 col-12 container-features">
                     <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Features</h5>
                     <ul class="list-group">
-
                         <a class="text-white mb-lg-3" href="https://www.codechef.com/ide" target="_blank">Code
                             Editor</a>
-                        <a class="text-white mb-lg-3" href="https://www.geeksforgeeks.org/data-structure-gq/"
+                        <a class="mb-lg-3" href="https://www.geeksforgeeks.org/data-structure-gq/"
                             target="_blank">Quizzes</a>
-                        <a class="text-white mb-lg-3" href="https://github.com/Algo-Phantoms/Algo-Tree"
+                        <a class="mb-lg-3" href="https://github.com/Algo-Phantoms/Algo-Tree"
                             target="_blank">DSA Guidance</a>
-                        <a class="text-white mb-lg-3" href="https://dare2compete.com/" target="_blank">Opportunities</a>
+                        <a class="mb-lg-3" href="https://dare2compete.com/" target="_blank">Opportunities</a>
 
                     </ul>
                 </div>
-                <div class="col-lg-3 col-md-3 col-sm-6 col-12">
+                <div class="col-lg-3 col-md-3 col-sm-6 col-12 Follow">
                     <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Follow Us</h5>
-                    <a href="#" target="_blank"><i class="fa fa-github p-2 text-white" aria-hidden="true"
-                            style="background:#3caea3; border-radius: 50%;"></i>
```

Comment:
Keep all other social media icons also.

Reply:
I'm thinking that instead of adding the social icon we can adding some important website links like term-condition, privacy policy, etc. because social icons have already been added to the side.

Reply:
That's good idea but let us keep that as a new issue and do instead of making the changes in this itself.

Reply:
Then can I PR, after remove unnecessary classes & hover effect on Image

Reply:
> Then can I PR, after remove unnecessary classes & hover effect on Image

Yes